.infrastructure/namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: todoapp
```

Comment:
![image](https://github.com/user-attachments/assets/5e95b9e0-fe70-4c6a-9e8a-6d323b537c1c)
Add Eof(End-of-file), need to add blank line at the end